config/routes.rb

```diff
@@ -891,6 +891,8 @@
 
     :ems_cloud                => {
       :get  => %w(
+        arbitration_profiles
```

Comment:
I do need this route in POST section for when redirecting back to list of Arbitration Profiles after add/save/cancel are pressed & in GET section for when actual link is clicked from the summary screen.